pkg/extensions/sync/destination.go

```diff
@@ -92,7 +92,7 @@ func (registry *DestinationRegistry) GetImageReference(repo, reference string) (
 func (registry *DestinationRegistry) CommitImage(imageReference types.ImageReference, repo, reference string) error {
 	imageStore := registry.storeController.GetImageStore(repo)
 
-	tempImageStore := getImageStoreFromImageReference(imageReference, repo, reference)
+	tempImageStore := getImageStoreFromImageReference(imageReference, repo, reference, registry.log)
```

Comment:
passing a logger around is useful

Reply:
Yes. But more importantly, loggers initialized as `log.Logger{}` does not ever actually output any logs. So the two loggers used here 

https://github.com/project-zot/zot/blob/e5eacaa082c9fa9f85fb4548841e8dc9f1918590/pkg/extensions/sync/destination.go#L304-L308

did never print anything to stdout. That means that logs such as https://github.com/project-zot/zot/blob/e5eacaa082c9fa9f85fb4548841e8dc9f1918590/pkg/storage/imagestore/imagestore.go#L73 would never fire